### PR TITLE
add images to reviews

### PR DESCRIPTION
**Pull request by livnekes**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2794,5 +2794,222 @@
...
+                                            </div>
+                                        </div>
+                                        <div class="yotpo-main">
+                                            <div class="content-title yotpo-font-bold"> VHS master cleanse letterpress </div>
+                                            <div class="yotpo-clr"></div>
+                                            <div class="yotpo-review-wrapper">
+                                                <div class="content-review"> Leggings before they sold out master cleanse, VHS direct trade you probably haven't heard of them ethical XOXO kogi street art Banksy. </div>
+                                            </div>
+                                            <div class="yotpo-review-images-wrapper">
+                                                <img width="84" height="84" alt="" class="image-review" src="http://placehold.it/84x84">
```
Comment (zanamich) [inline index.html]: @livnekes add code example

File: sass/core.scss [hunk 2]
```diff
@@ -1369,16 +1369,23 @@
...
         padding-bottom: 5px;
       }
-      .content-review{
+      .content-review {
         display: inline;
         color: $gray-darker;
       }
+      .yotpo-review-images-wrapper {
+        margin: 16px 0;
+        .image-review {
```
Comment (zanamich) [inline sass/core.scss:1384]: remove margin right for last child

**Comment by livnekes** [comment]
@zanamich done

**Comment by zanamich** [comment]
@gadinehab absolutely :+1:

**Comment by gadinehab** [comment]
@NetanelMoshkovitz ok

**Pull request merged**